src/plugin.ts

```diff
@@ -1,21 +1,8 @@
-import { Plugins } from '@capacitor/core';
+import { registerPlugin } from '@capacitor/core';
 
 import type { ISentryCapacitorPlugin } from './definitions';
 
-// Left here to support v2. This will be dropped in v4.
-// eslint-disable-next-line deprecation/deprecation
-const SentryCapacitor = Plugins.SentryCapacitor as ISentryCapacitorPlugin;
+const SentryCapacitor =
+  registerPlugin<ISentryCapacitorPlugin>('SentryCapacitor');
```

Comment:
How does this change affect supported capacitor version?

Reply:
Since capacitor 3, the old initialization code was deprecated (we are on capacitor 6). We no longer support capacitor 2 so it was a good time to rework how we retrieve the native reference.
The implemented method is supported from Capacitor 3 and it was validated on versions 3, 4, 5, 6

Reply:
Perfect, thank you for the context.